content/applications/hr/employees/departments.rst

```diff
@@ -0,0 +1,146 @@
+
+Click the :guilabel:`Unfold` button on a department card to expand it. Once expanded, the
+:guilabel:`Unfold` button changes to a :guilabel:`Fold` button. To collapse the department, click
+the :guilabel:`Fold` button. Only **one** department *per row* can be unfolded at a time.
+
+.. image:: departments/hierarchy.png
+ :align: center
+ :alt: The departments presented in a hierarchy view.
+
+Click anywhere on a department card to open the department form. Click the :guilabel:`(#) Employees`
+button to view a list of all the employees in that department, including all employees in the child
+departments beneath it, organized by individual department.
+
+.. example::
+   In the hierarchy view, if the :guilabel:`(#) Employees` button on the :guilabel:`Management` card
+   is clicked (the highest-level department card), **all** employees appear in a list view,
+   organized by department. This is because **all** departments are children of the
+   :guilabel:`Management` department.
+
+   If the :guilabel:`(#) Employees` button in the :guilabel:`Sales` department card is clicked, the
+   employees from the :guilabel:`Sales` department, as well as the two child departments
+   (:guilabel:`East Coast Territory` and :guilabel:`West Coat Territory`), appear in the list.
+
+   .. image:: departments/employee-list.png
+      :align: center
+      :alt: The list view of employees for the department that was clicked, including all child
+            departments.
+
+Create new departments
+======================
+
+To make a new department, navigate to :menuselection:`Employees app --> Departments`, then click
+:guilabel:`New` in the top-left to reveal a blank department form. Fill out the following
+information on the department form:
+
+.. image:: departments/department-form.png
+   :align: center
+   :alt: The department for with all fields filled out.
+
+- :guilabel:`Department Name`: enter a name for the department.
+- :guilabel:`Manager`: using the drop-down menu, select the department manager.
+- :guilabel:`Parent Department`: if the new department is housed within another department (has a
+  parent department), select the parent department using the drop-down menu.
+- :guilabel:`Custom Appraisal Templates`: if employees in this department require a specific
+  appraisal form that is different from the default appraisal form, tick the checkbox. If this
+  option is activated, an :guilabel:`Appraisal Survey` field appears on the form, and an
+  :guilabel:`Appraisal Templates` tab appears below the form. This field **only** appears if the
+  *Appraisals* app is installed, and the *360 Feedback* option is enabled in the settings. Refer to
+  the :doc:`../appraisals` documentation for more information.
+- :guilabel:`Company`: using the drop-down menu, select the company the department is part of.
+- :guilabel:`Color`: select a color for the department. Click the default white color box to display
```

Comment:
since you *do* mention/explain the 'Appraisal Templates' tab (which is only visible when the 'Custom Appraisal Templates' checkbox is ticked), you may want to consider briefly mentioning/explaining the 'Appraisal Survey' field -- which, actually, in the 17.0 runbot I am using, appears regardless of whether the checkbox for 'Custom Appraisal Templates' has been ticked. So, you'll probably want to conduct a quick double-check on that for accuracy during your revisions. Thanks! 👍